libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -612,10 +618,13 @@ class NavEKF3_core : public NavEKF_core_common
         EXTNAV=7        // Use external nav data
     };
 
-    // Specifies the rotation order used for the Tait-Bryan or Euler angles where alternative rotation orders are available
-    enum class rotationOrder {
-        TAIT_BRYAN_321=0,
-        TAIT_BRYAN_312=1
+    // specifies the method to be used when fusing yaw observations
+    enum class yawFusionMethod {
+	    MAGNETOMER=0,
```

Comment:
There's a "T" missing in MAGNETOMETER.  Beyond this I think we should try and reduce the use of synonyms because it adds unnecessary complexity.  I don't have strong feels of which word we should use (compass or magnetometer) but I think "Compass" is understood by more people than "Magnetometer" is.

Reply:
Ah, there are also some tabs which should be spaces.

Reply:
I would prefer to use the term 'magnetometer' within the EKF. It is technically more correct than the term 'compass' in this application (a sensor that measures a three axis magnetic flux, not a magnetic heading. 'Compass' should be limited to user facing interfaces.

Reply:
I've fixed the type.